lib/xcp_pci.ml

```diff
@@ -6,7 +6,7 @@ type address = {
 	dev: int;
 	fn: int;
 }
-[@@deriving sexp, rpc]
+[@@deriving sexp, rpc, rpcty]

```

Comment:
Do we use `rpc`? Can we just use `rpcty` instead, as done in other modules or itwill break lots of other code?

Reply:
I think that this is shared with the xenops_interface, which is still on camlp4.

Reply:
Non of the three things above is related to camlp4 though. I am expecting that removing rpc fails something, but I think it's worth double checking and see if we can reduce the autogenerated stuff if not needed

Reply:
Without it, xcpi-idl fails to build:
```
File "xen/xenops_types.ml", line 19, characters 2-224:
Error: Unbound value rpc_of_address
Hint: Did you mean typ_of_address?
make: *** [Makefile:7: release] Error 1
```

Reply:
Good, let's merge then